Update 04-pipefilter.md to reflect current episode structure (#1456)
swcarpentry/shell-novice: episodes/04-pipefilter.md

::::::::::::::::::::::::::::::::::::::: objectives

- Explain the advantage of linking commands with pipes and filters.

> - Combine commands in more than one stages to get new output

Reviewer: Perhaps use
```suggestion
- Combine sequences of commands to get new output
```

- Redirect a command's output to a file.
- Explain what usually happens if a program or pipeline isn't given any input to process.

::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::: questions

> - How can I produce various forms of the output of existing commands? How to show only part of the output? ( -> command line switches and pipes)

Reviewer: Perhaps use:
```suggestion
- How can I combine existing commands to produce a desired output?
- How can I show only part of the output? 
```

::::::::::::::::::::::::::::::::::::::: objectives

- Explain the advantage of linking commands with pipes and filters.
- Combine sequences of commands to get new output
- Redirect a command's output to a file.
- Explain what usually happens if a program or pipeline isn't given any input to process.

::::::::::::::::::::::::::::::::::::::::::::::::::

:::::::::::::::::::::::::::::::::::::::: questions

- How can I combine existing commands to produce a desired output?
- How can I show only part of the output? 

> - How can I combine existing commands to do new things? ( -> pipes)

Reviewer: This seems redundant, can possibly be removed.
```suggestion

```
